app/src/main/java/rick_and_morty/ui/episodes/EpisodesViewModel.kt

```diff
@@ -58,9 +58,4 @@ class EpisodesViewModel @Inject constructor(
     private fun saveEpisodesToDatabase(episodesList: List<EpisodeResultDto>) {
         realmInstance.saveEpisodesToDatabase(episodesList)
     }
```

Comment:
This also needs to be in `EpisodesRepository`. `RealmInstance` will not be injected into the view model. The idea is, that the view model does not know about the source of data (external/internal). The view model does only know about the repository.